tests/common/devices/multi_asic.py

```diff
@@ -856,3 +856,40 @@ def ports_list(self):
         """
         mg_facts = self.sonichost.minigraph_facts(host=self.sonichost.hostname)
         return list(mg_facts['ansible_facts']['minigraph_ports'].keys())
+
+    def get_admin_up_ports(self):
```

Comment:
@sdszhang Wondering if we can use the below API for this?
https://github.com/sonic-net/sonic-mgmt/blob/a11ea4974c48ddef612574439fb74f6ddee2d29c/tests/common/platform/interface_utils.py#L14C5-L14C22